ledger/storage/object/indexsaver.go

```diff
@@ -0,0 +1,60 @@
+package object
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/insolar/insolar/insolar"
+	"github.com/insolar/insolar/insolar/message"
+	"github.com/insolar/insolar/insolar/reply"
+	"github.com/pkg/errors"
+)
+
+//go:generate minimock -i github.com/insolar/insolar/ledger/storage/object.IndexSaver -o ./ -s _mock.go
+
+type IndexSaver interface {
```

Comment:
We don't need this component. We have `proc.GetIndex`.

Reply:
I'm not 100% convinced that rewriting this code to proc.GetIndex Procedure in this task is a good idea. It will affects other message handlers that currently use IndexSaver.SaveIndexFromHeavy (ex saveIndexFromHeavy). We can create a task for this refactoring though if you like.